draft-ietf-core-coap-tcp-tls.md

```diff
@@ -1267,7 +1276,7 @@ established with {{-urireg}}.
 
 ### coap+ws

```

Comment:
Yes, please, but preferably when we have stabilized -03, as this conversion is a wholesale change that makes finding diffs harder (need to run diff with ignore-whitespace, and git blame can't do that, etc.).